app/client-app/src/app/pages/Editor/components/PageList/DraggablePageItem.jsx

```diff
@@ -94,6 +95,36 @@ const QuestionButton = ({ isQuestionItem, id, setQuestionItem, type }) => {
   );
 };
 
+const OptionalButton = ({ isOptional, id, setIsOptional }) => {
+  const handleOptionalClick = () => setIsOptional(id, !isOptional);
+  const { busy } = usePageListContext();
+
+  if (some(busy)) {
+    const p = { p: "9px", dotSize: "14px" };
+    return (
+      <Center width={8}>
+        <PlaceholderDot {...p} />
+      </Center>
+    );
+  }
+
+  return (
+    <Tooltip label={isOptional ? "Make Mandotary" : "Make Optional"}>
```

Comment:
```suggestion
    <Tooltip label={isOptional ? "Make Mandatory" : "Make Optional"}>
```
small but because it's user facing ;)